.github/actions/install-aiida-core/action.yml

```
@@ -0,0 +1,50 @@
+name: Install aiida-core
+description: Install aiida-core package and its Python dependencies
+
+inputs:
+  python-version:
+    description: Python version
+    default: '3.9'              # Lowest supported version
+    required: false
+  extras:
+    description: aiida-core extras (including brackets)
+    default: ''
+    required: false
+  # NOTE: Hard-learned lesson: we cannot use type=boolean here, apparently :-(
+  # https://stackoverflow.com/a/76294014
+  from-requirements:
+    description: Install aiida-core dependencies from pre-compiled requirements.txt file
+    default: 'true'
+    required: false
+
+runs:
+  using: composite
+  steps:
+  - name: Set Up Python 🐍
```

I'm allergic to emojis. Don't tell Marnik
```suggestion
  - name: Set Up Python
```